src/emails/templates/boilerplate/BoilerplateEmail.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
We discussed already that this can't `use client` or `use server` since for Storybook we need client support but the emails will always be rendered by the server. Email is unusual in that it's just a static page we're generating, there is no client or server runtime, so just including an existing component used by the website is unlikely to work.

Could we have a comment here to that effect? Too bad there is no `use email` ;)

Reply:
Good callout, added in e6b8d96c6.